lib/model/ui/user/ui_child.dart

```diff
@@ -8,11 +8,25 @@ class UIChild extends UIUser {
 	final int todayPlanCount;
 	final bool hasActivePlan;
 	final Map<CurrencyType, int> points;
+	final Map<CurrencyType, String> pointsNames;
```

Comment:
Hmm, that originates from quite a while back, should probably be replaced with just `List<UIPoints>`